.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+    "build": {
+        "context": ".",
+        "dockerfile": "Dockerfile"
+    },
+    
+    // Features to add to the dev container. More info: https://containers.dev/features
+    "features": {
+        "ghcr.io/devcontainers/features/nvidia-cuda:1": {
```

Comment:
This Feature takes approx two mins to run and is only useful for GPU powered codespaces.

Not sure, if we should comment this and ask the user to uncomment & rebuild only if he has a GPU codespace.
Not sure what experience we'd like here.

1.  comment out as building container is 2 mins faster & majority users will use CPU codespaces
2. Keep it uncommented to avoid the user to rebuild a codespace which will take double the wait time 

Note: The entire container takes 5 min.

Reply:
I'll update the docs accordingly

Reply:
I say comment it out until GPUs are generally available in Codespaces. Definitely need to document that there are notebooks that won't run without the GPU

Reply:
Makes sense 👍